src/register/RegistrationPage.jsx

```diff
@@ -527,7 +529,12 @@ const RegistrationPage = (props) => {
             <Spinner animation="border" variant="primary" id="tpa-spinner" />
           </div>
         ) : (
-          <div className="mw-xs mt-3">
+          <div
+            className={classNames(
+              'mw-xs mt-3',
+              { 'form--centerd': variant },
```

Comment:
We can use paragon classes for this
```suggestion
              { 'w-100 m-auto': isRegistrationEmbedded },
```